### ISSUE TITLE
Clearer Navigation

### ISSUE THREAD

**Issue by Illyism** [musicplayer-io/redditmusicplayer#27]
> It would awesome if some of the symbols have tool tips or maybe a help menu to help us navigate :D

**Comment by KhalidGit** [comment]
Are you looking for something like a styled version of the jQuery Tooltip?

**Comment by Illyism** [comment]
Nah, Semantic UI has a [popup](http://semantic-ui.com/modules/popup.html) element. I'm already using it for the GitHub link in the titlebar.

**Comment by KhalidGit** [comment]
And are you wanting a click event or just a hover? Haven't used Semantic before, but I'll figure it out.

**Comment by Illyism** [comment]
Sorry for not being clear here. I simply copy-pasted a comment from Reddit.

The underlying issue is that the navigation is not so clear on what to do. I'm using icons to make it more well decorated and to leave out space for the other components but the truth is that it's not so easy to hover on an mobile device either so mobile users won't have any tooltips.

A guide on how to use it could help but I'd rather have the interface be easy to understand by its own.

Something like Slack does it is maybe something we could do, explain each interface step-by-step.
